darwin/importer/formats/labelbox_schemas.py

```diff
@@ -0,0 +1,78 @@
+bounding_box = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://darwin.v7labs.com/schemas/bounding_box",
+    "description": "Schema of a Bounding Box",
+    "title": "Bounding Box",
+    "default": {"top": 1.2, "left": 2.5, "height": 10, "width": 20},
+    "examples": [{"top": 0, "left": 0, "height": 10, "width": 20}],
+    "type": "object",
+    "properties": {
+        "top": {"type": "number"},
+        "left": {"type": "number"},
+        "height": {"type": "number"},
+        "width": {"type": "number"},
+    },
+    "required": ["top", "left", "height", "width"],
+}
+
+point = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://darwin.v7labs.com/schemas/point",
+    "description": "Schema of a Point",
+    "title": "Point",
+    "default": {"x": 1.2, "y": 2.5},
+    "examples": [{"x": 1.2, "y": 2.5}, {"x": 0, "y": 1}],
+    "type": "object",
+    "properties": {"x": {"type": "number"}, "y": {"type": "number"}},
+    "required": ["x", "y"],
+}
+
+polygon = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://darwin.v7labs.com/schemas/polygon",
+    "description": "Schema of a Polygon",
+    "title": "Polygon",
+    "default": [{"x": 1.2, "y": 2.5}, {"x": 2.5, "y": 3.6}, {"x": 1.2, "y": 2.5}],
+    "examples": [[{"x": 1.2, "y": 2.5}, {"x": 2.5, "y": 3.6}, {"x": 1.2, "y": 2.5}], []],
+    "type": "array",
+    "items": point,
+}
+
+
+label_object = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": ["title"],
+    "properties": {"title": {"type": "string"}},
+    "oneOf": [
+        {"type": "object", "required": ["point"], "properties": {"point": point}},
```

Comment:
You don't need the `"type": "object"` here as it's already included in the level above.

Reply:
I am specifying that the items of the array of of type "object". The above "object" type is for the father object.
Unless there is an implicit rule I am not aware of, I think this is correct.

Reply:
This isn't how it works. Whatever you specify under `oneOf` gets merged with the level above. So here the level above specifies `type: object` and alternatives under `oneOf` specify the same which result in... `type: object`. Let's take an example (pseudo-code for brevity):

```yaml
type: object
properties:
  title:
    type: string
oneOf:
  - properties:
      one:
        type: string
    required: one
  - properties:
      two:
        type: string
    required: two
```
Makes it resolve to these two alternative schemas:

```yaml
type: object
properties:
  title:
    type: string
  one:
    type: string
required: one
```
and:
```yaml
type: object
properties:
  title:
    type: string
  two:
    type: string
required: two
```

... which are tried one after another by the validator. Notice that adding `type: object` to children of `oneOf` wouldn't change the resulting alternative schemas at all.

Hope this helps.

Reply:
Fixed !